### DISCUSSION TITLE
How to remove blog section

### DISCUSSION THREAD

**Discussion by images2symbols** [alshedivat/al-folio#627] [Q&A] [answered]
I don't mean just removing it from the navbar, I don't want https://site.github.io/blog/ to be a valid page

**Comment by soheil-h** [comment, marked as answer]
In addition to removing lines related to "blog" in header.html, I removed lines 85-95 in _config.yml

**Comment by rohandebsarkar** [comment]
Delete the `blog` folder at the root directory.

**Comment by Benhrpr** [comment]
The 'latest posts' text on the landing page is still hyperlinked to /blog/, does anyone know where the code that controls this is? @rohandebsarkar @soheil-h

**Reply by george-gca** [reply]
It is in the page itself, in this case `_pages/about.md` frontmatter.

**Comment by victorballester7** [comment]
If I do this

> In addition to removing lines related to "blog" in header.html, I removed lines 85-95 in _config.yml

I still get something, as shown in the photo below:

![image](https://github.com/alshedivat/al-folio/assets/78110444/e3e4b0e2-a0e3-42bc-ac74-48e0ae1d71cd)

What I want is to redirect exactly all the pages except from a few wants (`cv` and `about`) to the following page:

![image](https://github.com/alshedivat/al-folio/assets/78110444/3a1fffba-03ec-4103-84f8-d43702708ce5)

I managed to do that with the pages under the folder `pages`, like `repositories` or `publications` by uncommenting the permalink line in their respective markdown files, but I'm struggling with `blog` and `news`. And related to that, how can I know which are ALL the subpages of my website? Because maybe there are more (and indeed they are, like `sample_posts` that I have just discovered).

Thank you! ❤️

**Comment by george-gca** [comment]
> Delete the `blog` folder at the root directory.

Have you tried also doing this?

For the news case, you can delete the lines 166-170 from your [_config.yml](https://github.com/alshedivat/al-folio/blob/85f2590a2119ec57063845d628221155044ebea8/_config.yml#L166) related to the `news` collection, and also delete the file [news.html](https://github.com/alshedivat/al-folio/blob/master/news.html) from the root directory.

**Comment by victorballester7** [comment]
Ok, first, thank you so much for your rapid comment!! 😄 

You were right with the `news` tab. In fact, I manage to understand a little bit more about how the page works (I am a newbie in html) and deleting the folder of `posts` and  `blog`  completely removes the `blog` tab, having the appropriate section of `blog` commented in the `_config.yml` file.
